readme.md

```diff
@@ -52,9 +52,10 @@ There are two numbers that go at the bottom of the file. The first is the number
 ### Configuration
 There is a configuration file `config.json` where you can edit the bot's settings.
 
-`movesShown` The number of top moves that are shown.<br>
-`gemManagement` Toggles a gem management algorithm that when enabled will recommend moves that score less to collect more gems if it thinks you'll likely profit in the long term.<br>
-`prettyPrint` For each move, shows a 5x5 board that highlights the move. Swaps are marked in red.
+- `movesShown` The number of top moves that are shown.<br>
+- `gemManagement` Toggles a gem management algorithm that when enabled will recommend moves that score less to collect more gems if it thinks you'll likely profit in the long term.<br>
+- `prettyPrint` For each move, shows a 5x5 board that highlights the move. Swaps are marked in red.
+- `multiProcessing` Uses all cpus of your processor to significantly speed-up the solver. Makes a huge difference if 3 swaps are required. Please ensure you have enough RAM available, as more CPU usage typically requires more RAM.
```

Comment:
Please may I suggest the config option description:
Uses multiple processor cores to compute moves.
I think something a little shorter is more appropriate

Reply:
Sure, updated